features/nanostack/FEATURE_NANOSTACK/mbed-mesh-api/source/thread_tasklet.c

```diff
@@ -282,14 +290,23 @@ void thread_tasklet_configure_and_connect_to_network(void)
     ns_dyn_mem_free(device_configuration.PSKd_ptr);
     device_configuration.PSKd_ptr = (uint8_t*)dyn_buf;
     device_configuration.PSKd_len = sizeof(PSKd) - 1;
-
-    thread_tasklet_data_ptr->link_config.key_rotation = 3600;
-    thread_tasklet_data_ptr->link_config.key_sequence = 0;
-
-    thread_management_node_init(thread_tasklet_data_ptr->nwk_if_id,
+    
+    bool is_static_config = MBED_MESH_API_THREAD_STATIC_LINK_CONFIG;    
```

Comment:
should you make link configuration pointer and either set it to NULL or correct value based on flag